ynr/apps/data_exports/filters.py

```diff
@@ -47,7 +47,17 @@ def filter(self, qs, value):
         return qs
 
 
-class PartyINFilter(BaseInFilter, django_filters.CharFilter):
+class DateIncludingReplacedFilter(django_filters.CharFilter):
+    def filter(self, qs, value):
+        if value:
+            qs = qs.filter(
+                Q(election_date=value)
+                | Q(ballot_paper__replaces__ballot_paper_id__contains=value)
+            )
+        return qs
+
+
+class PartyINFilter(django_filters.MultipleChoiceFilter):
     pass
```

Comment:
Two things here:
- Just for my own understanding, why is this called `PartyINFilter`? What is IN in this context?
- Why define an empty filter here and then define the `field_name` and `choices` inline. Wouldn't it make more sense to define those here?

Reply:
I think this is a hang over from when it used to be `PartyINFilter(BaseInFilter, django_filters.CharFilter)`. I'll update the field to use `MultipleChoiceFilter` directly. :+1: